llpc/test/shaderdb/general/PipelineVsFs_TestPrimitiveID_First.pipe

```diff
@@ -0,0 +1,73 @@
+; Test primitiveID when provoking vertex mode is FIRST
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v --gfxip=10.1.0 %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST: %{{[0-9]*}} = lshr i32 %{{[^,]*}}, 2
+; SHADERTEST: %{{[0-9]*}} = and i32 %{{[0-9]*}}, 2044
```

Comment:
This test seems pretty generic.  It seems likely that other instruction could be matched, and this will still pass.  The significance of these instructions is not clear to me.  It might be more useful to other developers if you could add more context.  

1. Add a line to indicate which function these should be in. 
2. Use the [string substitution blocks](https://llvm.org/docs/CommandGuide/FileCheck.html#filecheck-string-substitution-blocks) to explain the connection between the two instructions.
3. If the expression must come from a particular parameter or must be used as an index to load from an array, adding the extra instructions to make sure that is happening will be useful.